docs/src/verification.md

```diff
@@ -0,0 +1,164 @@
+# Verification
+
+## General Overview
+
+When comparing our `compiled_model` with the `framework_model` (e.g., `PyTorch`), we aim to verify whether the output from the `compiled_model` is sufficiently similar to the output from the `framework_model` (where required degree of similarity is configurable).
+
+So generally we want to perform the following steps:
+1. Create a framework model
+2. Run a forward pass through the framework model
+3. Compile the framework model using `Forge`
+4. Run a forward pass through the compiled model
+5. Compare the outputs.
+
+Most of the above steps `verify()` function does for us:
+- Handles forward passes for both framework and compiled models
+- Compares results using a combination of comparison methods
+- Supports customization through the `VerifyConfig` class.
+
+**Example of usage**
+```python
+def test_add():
+
+	class Add(nn.Module):
+		def __init__(self):
```

Comment:
fix indent (4 spaces)

Reply:
for some reason github renders it to 8 spaces, even though it's 4 spaces for me locally.

Reply:
you should use spaces, not tabs